GVRf/Framework/src/org/gearvrf/GVRMesh.java

```diff
@@ -153,8 +153,8 @@ public void setTexCoords(float[] texCoords) {
      *            Array containing the packed triangle index data.
      */
     public void setTriangles(char[] triangles) {
-        isNonNull(triangles);
-        isValidData(triangles.length, 3);
+        checkNotNull("triangle", triangles);
```

Comment:
This is why it would be nice if Java had a macro to turn an identifier into a string: `"triangle"` s/b `"triangles"`.